resources/views/layouts/base.blade.php

```diff
@@ -37,9 +37,14 @@
     <link href="/css/base.css" rel="stylesheet">
     <link href="/css/toastr.min.css" rel="stylesheet">
     <link href="/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.css">
 
     <link rel="shortcut icon" href="favicon.ico">
     @yield('css')
+
+    <script src="/js/jquery-1.11.3.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
```

Comment:
@cydrobolt Moved to the end of the body. In fact using CDN can speed up the page, depending where it is hosted and where is the user. CDNs usually has edge locations to be able to deliver content as fast as possible. So I suggest to leave it as it is. There are ale other pros of such solution like maximum concurrency connections to same domain - that is why domain sharding is frequently introduced.